README.md

```diff
@@ -69,11 +69,13 @@ Usage
 If you are happy with the build defaults, then you can use the openresty image from the [Docker Hub](https://hub.docker.com/r/openresty/openresty/).  The image tags available there are listed at the top of this README.
 
 ```
-docker run [options] openresty/openresty:latest-trusty /usr/local/openresty/nginx/sbin/nginx
+docker run [options] openresty/openresty:latest-trusty"
 ```
 
 *[options]* would be things like -p to map ports and -v to map volumes.
 
+The `-g "daemon off;"` flag is used in the entrypoint to keep the nginx daemon running after container creation, this flag can also be added to the nginx.conf and ommitted from the entrypoint.
```

Comment:
The `-g "daemon off;"` directive is used in the Dockerfile ENTRYPOINT to keep the Nginx daemon running after container creation.  This directive can also be added to the nginx.conf and omitted from the ENTRYPOINT.

To invoke with another ENTRYPOINT, for example the `resty` utility, invoke like so:
```
docker run [options] --entrypoint /usr/local/openresty/bin/resty openresty/openresty:latest-alpine [script.lua]
```